docs/reference-metrics.md

```diff
@@ -140,7 +140,8 @@ All the broker metrics are labeled with the following labels:
 | pulsar_ml_cache_pool_used | Gauge  | The total used memory of chunk lists in direct arena. |
 | pulsar_ml_cache_used_size | Gauge  | The size used to store the payloads of entries (in bytes). |
 | pulsar_ml_count | Gauge  | The number of currently opened managed ledgers. |
-| topic_load_times | Summary | The topic load latency calculated in milliseconds. |
+| ~~topic_load_times (deprecated)~~ | ~~Summary~~ | ~~The topic load latency calculated in milliseconds.~~ |
```

Comment:
I add strikethroughs on purpose (to make it more prominent and allow users to be aware of it's deprecated).